Problem building datagen-base image

*Problem:*
Cannot build datagen-base image. 

*Suspected cause:*
Missing maven dependencies

*To reproduce:*
Start in the root directory of the project (branch: main)
Run:
```
cd base-docker-image/
docker build . --tag ldbc/datagen_base:
```

Tail of the execution log:
```

...

:: loading settings :: url = jar:file:/root/.m2/repository/org/walkmod/walkmod-cmd/3.0.0/walkmod-cmd-3.0.0.jar!/ivysettings.xml
:: loading settings :: url = jar:file:/root/.m2/repository/org/apache/ivy/ivy/2.4.0-rc1/ivy-2.4.0-rc1.jar!/org/apache/ivy/core/settings/ivysettings.xml
:: resolving dependencies :: org.walkmod#walkmod-pmd-plugin-caller;working
        confs: [default]
:: resolution report :: resolve 11745ms :: artifacts dl 0ms
        ---------------------------------------------------------------------
        |                  |            modules            ||   artifacts   |
        |       conf       | number| search|dwnlded|evicted|| number|dwnlded|
        ---------------------------------------------------------------------
        |      default     |   3   |   0   |   0   |   0   ||   0   |   0   |
        ---------------------------------------------------------------------

:: problems summary ::
:::: WARNINGS
                module not found: org.walkmod#walkmod-pmd-plugin;latest.integration

        ==== maven2: tried

          http://repo1.maven.org/maven2/org/walkmod/walkmod-pmd-plugin/[revision]/walkmod-pmd-plugin-[revision].pom

          -- artifact org.walkmod#walkmod-pmd-plugin;latest.integration!walkmod-pmd-plugin.jar:

          http://repo1.maven.org/maven2/org/walkmod/walkmod-pmd-plugin/[revision]/walkmod-pmd-plugin-[revision].jar

        ==== jboss: tried

          -- artifact org.walkmod#walkmod-pmd-plugin;latest.integration!walkmod-pmd-plugin.jar:

          http://repository.jboss.com/maven2/org/walkmod/walkmod-pmd-plugin/[revision]/walkmod-pmd-plugin-[revision].jar

        ==== springsource-external: tried

          http://repository.springsource.com/maven/bundles/external/org/walkmod/walkmod-pmd-plugin/[revision]/walkmod-pmd-plugin-[revision].pom

          -- artifact org.walkmod#walkmod-pmd-plugin;latest.integration!walkmod-pmd-plugin.jar:

          http://repository.springsource.com/maven/bundles/external/org/walkmod/walkmod-pmd-plugin/[revision]/walkmod-pmd-plugin-[revision].jar

        ==== springsource-release: tried

          http://repository.springsource.com/maven/bundles/release/org/walkmod/walkmod-pmd-plugin/[revision]/walkmod-pmd-plugin-[revision].pom

          -- artifact org.walkmod#walkmod-pmd-plugin;latest.integration!walkmod-pmd-plugin.jar:

          http://repository.springsource.com/maven/bundles/release/org/walkmod/walkmod-pmd-plugin/[revision]/walkmod-pmd-plugin-[revision].jar

        ==== springio-release: tried

          http://repo.spring.io/libs-release-remote/org/walkmod/walkmod-pmd-plugin/[revision]/walkmod-pmd-plugin-[revision].pom

          -- artifact org.walkmod#walkmod-pmd-plugin;latest.integration!walkmod-pmd-plugin.jar:

          http://repo.spring.io/libs-release-remote/org/walkmod/walkmod-pmd-plugin/[revision]/walkmod-pmd-plugin-[revision].jar

                module not found: org.walkmod#walkmod-maven-plugin;latest.integration

        ==== maven2: tried

          http://repo1.maven.org/maven2/org/walkmod/walkmod-maven-plugin/[revision]/walkmod-maven-plugin-[revision].pom

          -- artifact org.walkmod#walkmod-maven-plugin;latest.integration!walkmod-maven-plugin.jar:

          http://repo1.maven.org/maven2/org/walkmod/walkmod-maven-plugin/[revision]/walkmod-maven-plugin-[revision].jar

        ==== jboss: tried

          -- artifact org.walkmod#walkmod-maven-plugin;latest.integration!walkmod-maven-plugin.jar:

          http://repository.jboss.com/maven2/org/walkmod/walkmod-maven-plugin/[revision]/walkmod-maven-plugin-[revision].jar

        ==== springsource-external: tried

          http://repository.springsource.com/maven/bundles/external/org/walkmod/walkmod-maven-plugin/[revision]/walkmod-maven-plugin-[revision].pom

          -- artifact org.walkmod#walkmod-maven-plugin;latest.integration!walkmod-maven-plugin.jar:

          http://repository.springsource.com/maven/bundles/external/org/walkmod/walkmod-maven-plugin/[revision]/walkmod-maven-plugin-[revision].jar

        ==== springsource-release: tried

          http://repository.springsource.com/maven/bundles/release/org/walkmod/walkmod-maven-plugin/[revision]/walkmod-maven-plugin-[revision].pom

          -- artifact org.walkmod#walkmod-maven-plugin;latest.integration!walkmod-maven-plugin.jar:

          http://repository.springsource.com/maven/bundles/release/org/walkmod/walkmod-maven-plugin/[revision]/walkmod-maven-plugin-[revision].jar

        ==== springio-release: tried

          http://repo.spring.io/libs-release-remote/org/walkmod/walkmod-maven-plugin/[revision]/walkmod-maven-plugin-[revision].pom

          -- artifact org.walkmod#walkmod-maven-plugin;latest.integration!walkmod-maven-plugin.jar:

          http://repo.spring.io/libs-release-remote/org/walkmod/walkmod-maven-plugin/[revision]/walkmod-maven-plugin-[revision].jar

                module not found: org.walkmod#walkmod-javalang-plugin;[2.0, 4.0)

        ==== maven2: tried

          http://repo1.maven.org/maven2/org/walkmod/walkmod-javalang-plugin/[revision]/walkmod-javalang-plugin-[revision].pom

          -- artifact org.walkmod#walkmod-javalang-plugin;[2.0, 4.0)!walkmod-javalang-plugin.jar:

          http://repo1.maven.org/maven2/org/walkmod/walkmod-javalang-plugin/[revision]/walkmod-javalang-plugin-[revision].jar

        ==== jboss: tried

          -- artifact org.walkmod#walkmod-javalang-plugin;[2.0, 4.0)!walkmod-javalang-plugin.jar:

          http://repository.jboss.com/maven2/org/walkmod/walkmod-javalang-plugin/[revision]/walkmod-javalang-plugin-[revision].jar

        ==== springsource-external: tried

          http://repository.springsource.com/maven/bundles/external/org/walkmod/walkmod-javalang-plugin/[revision]/walkmod-javalang-plugin-[revision].pom

          -- artifact org.walkmod#walkmod-javalang-plugin;[2.0, 4.0)!walkmod-javalang-plugin.jar:

          http://repository.springsource.com/maven/bundles/external/org/walkmod/walkmod-javalang-plugin/[revision]/walkmod-javalang-plugin-[revision].jar

        ==== springsource-release: tried

          http://repository.springsource.com/maven/bundles/release/org/walkmod/walkmod-javalang-plugin/[revision]/walkmod-javalang-plugin-[revision].pom

          -- artifact org.walkmod#walkmod-javalang-plugin;[2.0, 4.0)!walkmod-javalang-plugin.jar:

          http://repository.springsource.com/maven/bundles/release/org/walkmod/walkmod-javalang-plugin/[revision]/walkmod-javalang-plugin-[revision].jar

        ==== springio-release: tried

          http://repo.spring.io/libs-release-remote/org/walkmod/walkmod-javalang-plugin/[revision]/walkmod-javalang-plugin-[revision].pom

          -- artifact org.walkmod#walkmod-javalang-plugin;[2.0, 4.0)!walkmod-javalang-plugin.jar:

          http://repo.spring.io/libs-release-remote/org/walkmod/walkmod-javalang-plugin/[revision]/walkmod-javalang-plugin-[revision].jar

                ::::::::::::::::::::::::::::::::::::::::::::::

                ::          UNRESOLVED DEPENDENCIES         ::

                ::::::::::::::::::::::::::::::::::::::::::::::

                :: org.walkmod#walkmod-pmd-plugin;latest.integration: not found

                :: org.walkmod#walkmod-maven-plugin;latest.integration: not found

                :: org.walkmod#walkmod-javalang-plugin;[2.0, 4.0): not found

                ::::::::::::::::::::::::::::::::::::::::::::::


:::: ERRORS
        SERVER ERROR: HTTPS Required url=http://repo1.maven.org/maven2/org/walkmod/walkmod-pmd-plugin/maven-metadata.xml

        SERVER ERROR: HTTPS Required url=http://repo1.maven.org/maven2/org/walkmod/walkmod-pmd-plugin/

        SERVER ERROR: HTTPS Required url=http://repo1.maven.org/maven2/org/walkmod/walkmod-pmd-plugin/

        SERVER ERROR: HTTPS Required url=http://repo1.maven.org/maven2/org/walkmod/walkmod-pmd-plugin/

        SERVER ERROR: HTTPS Required url=http://repo1.maven.org/maven2/org/walkmod/walkmod-pmd-plugin/

        SERVER ERROR: HTTPS Required url=http://repo1.maven.org/maven2/org/walkmod/walkmod-maven-plugin/maven-metadata.xml

        SERVER ERROR: HTTPS Required url=http://repo1.maven.org/maven2/org/walkmod/walkmod-maven-plugin/

        SERVER ERROR: HTTPS Required url=http://repo1.maven.org/maven2/org/walkmod/walkmod-maven-plugin/

        SERVER ERROR: HTTPS Required url=http://repo1.maven.org/maven2/org/walkmod/walkmod-maven-plugin/

        SERVER ERROR: HTTPS Required url=http://repo1.maven.org/maven2/org/walkmod/walkmod-maven-plugin/

        SERVER ERROR: HTTPS Required url=http://repo1.maven.org/maven2/org/walkmod/walkmod-javalang-plugin/maven-metadata.xml

        SERVER ERROR: HTTPS Required url=http://repo1.maven.org/maven2/org/walkmod/walkmod-javalang-plugin/

        SERVER ERROR: HTTPS Required url=http://repo1.maven.org/maven2/org/walkmod/walkmod-javalang-plugin/

        SERVER ERROR: HTTPS Required url=http://repo1.maven.org/maven2/org/walkmod/walkmod-javalang-plugin/

        SERVER ERROR: HTTPS Required url=http://repo1.maven.org/maven2/org/walkmod/walkmod-javalang-plugin/


:: USE VERBOSE OR DEBUG MESSAGE LEVEL FOR MORE DETAILS
 WARN [main] - unresolved dependency: org.walkmod#walkmod-pmd-plugin;latest.integration: not found
 WARN [main] - unresolved dependency: org.walkmod#walkmod-maven-plugin;latest.integration: not found
 WARN [main] - unresolved dependency: org.walkmod#walkmod-javalang-plugin;[2.0, 4.0): not found
ERROR [main] - /opt/ldbc_snb_datagen/walkmod.xml is invalid. Please, execute walkmod with -e to see the details.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:25 min
[INFO] Finished at: 2021-12-06T09:47:56+00:00
[INFO] Final Memory: 21M/192M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.walkmod.maven.plugins:walkmod-maven-plugin:1.0.3:apply (default) on project ldbc_snb_datagen: Execution default of goal org.walkmod.maven.plugins:walkmod-maven-plugin:1.0.3:apply failed. NullPointerException -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
Error: error building at STEP "RUN mvn -DskipTests clean assembly:assembly": error while running runtime: exit status 1
```

--- a/base-docker-image/pom.xml
+++ b/base-docker-image/pom.xml
@@ -95,23 +95,6 @@
         </configuration>
       </plugin>
       <plugin>
-        <groupId>org.walkmod.maven.plugins</groupId>
-        <artifactId>walkmod-maven-plugin</artifactId>
-        <version>1.0.3</version>
-        <executions>
-          <execution>
-            <phase>generate-sources</phase>
-            <goals>
-              <goal>apply</goal>
-            </goals>
-            <configuration>
-              <chains>pmd</chains>
-              <properties>configurationFile=rulesets/java/basic.xml</properties>
-            </configuration>
-          </execution>
-        </executions>
-      </plugin>
-      <plugin>
         <artifactId>maven-assembly-plugin</artifactId>
         <version>2.4</version>
         <configuration>

